api/app/zonal_stats.py

```diff
@@ -257,6 +269,7 @@ def calculate_stats(
 
     if filter_by is not None:
         key, value = filter_by
+        # TODO - return properties as well. They are needed in the frontend for grouping
```

Comment:
@JorgeMartinezG please have a look at this and my change line 284. The changes in https://github.com/WFP-VAM/prism-app/pull/616 broke analysis table generation in the frontend because it removed necessary "properties".

cc @wadhwamatic